bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -223,6 +224,25 @@ impl Client {
         Ok(device_id.to_string())
     }
 
+    /// Get an event of the given type out of the account data store. The event
+    /// will be returned as a JSON string.
+    pub fn get_account_data_event(&self, event_type: String) -> anyhow::Result<Option<String>> {
+        RUNTIME.block_on(async move {
+            let event = self.store().get_account_data_event(event_type.into()).await?;
```

Comment:
I added this before `get_account_data_raw` was added to the account. Can update to use that method instead if it makes more sense.

Reply:
I think it does, yeah.

Reply:
Updated (and slightly renamed the method).

While we're looking at this one, would there be a way for this method to return only the value of `content` as the JSON string? I've made the iOS `ClientProxy` decode the whole event and then return just the content but would be nice to do this in the SDK for all clients if it's somehow possible.

Reply:
Yeah, that would be possible and I really like the idea. I'll look into it tomorrow.